app/services/base_logger.js

```diff
@@ -0,0 +1,43 @@
+class Logger {
+  constructor(bunyanLogger) {
+    this.bunyanLogger = bunyanLogger;
+    this.children = [];
+  }
+
+  child(opts) {
```

Comment:
In witch cases do we need to call a child logger? Would you might providing some information about that?

Reply:
![image](https://user-images.githubusercontent.com/9417851/38237373-bd7c24d8-372f-11e8-8822-b32b63ebc9e3.png)
In this case, child logger is created with the specific name and when writing f.e `log.info` in the console we can see this `component` field and understand were this log was emitted

Reply:
You can check it. [Link](https://github.com/inst-event-emitter/eventemitter-backend/pull/7/commits/ad112cb8c1d9c64199e855f153ea3258174bfc77#diff-586cf549a824167942b3b214a6303c48R29)